### ISSUE TITLE
Upload failed without flags args

### ISSUE THREAD

**Issue by shoma** [codecov/codecov-circleci-orb#2]
The default var for flags is empty string. But if omit this, upload failed.

## My .circleci.yml (partial)
```
      - codecov/upload:
          file: build/coverage.txt
```

## Packed orb config  (partial)
```yaml
   # Orb 'codecov/codecov@1.0.1' resolved to 'codecov/codecov@1.0.1'
    - run:
        name: Upload Coverage Results
        command: |
          bash <(curl -s https://codecov.io/bash) \
            -F  \
            -f build/coverage.txt \
            -n ${CIRCLE_BUILD_NUM} \
            -t ${CODECOV_TOKEN} \
            -y .codecov.yml \
```

## CI Log (partial)
```
==> Uploading reports
    url: https://codecov.io
    query: branch=my-branch&commit=<hash>&build=1000&build_url=&name=&tag=&slug=shoma%2Fawesomerepo&service=circleci&flags=-f&pr=&job=0
    -> Pinging Codecov
    -> Uploading
    X> Failed to upload
```

The flags field have `-f` in URL Query part.

**Comment by thomasrockhu** [comment]
Hi @shoma thanks for catching this!  I'll have the team take a look today.

**Comment by thomasrockhu** [comment]
@shoma this should have been fixed with the new release `1.0.2`.  I'm going to close out this issue, but feel free to reopen if it doesn't solve your problem.
